modules/s3db/budget.py

```diff
@@ -1299,13 +1299,12 @@ def model(self):
                                           writable = True,
                                           represent = self.budget_CostItemRepresent(),
                                           ),
-                          # @ToDo: s3_datetime
-                          s3_date("start_date",
-                                  label = T("Start Date")
-                                  ),
-                          s3_date("end_date",
-                                  label = T("End Date")
-                                  ),
+                          s3_datetime("start_date",
```

Comment:
I would be very wary of accepting this as-is.
As-is this is obviously trivial....so why was it left as a ToDo?
Have you been through the module & tested that all still works with datetimes?
- ideally writing some Tests could help with proving this.

UX-wise, I would think that we would want to enter dates in most cases as is done now, rather than a datetime widget (look for the s3_datetime option to allow this)

There is also potentially a data migration issue...I'm not personally familiar with the issues when moving dates to datetimes. I'm not too worried about this personally as I'm not aware of the module being used in Production yet, however in general it is somethign to be considered when making such changes.